modules/raycast/raycast_occlusion_cull.cpp

```diff
@@ -314,10 +314,10 @@ void RaycastOcclusionCull::scenario_remove_instance(RID p_scenario, RID p_instan
 }
 
 void RaycastOcclusionCull::Scenario::_update_dirty_instance_thread(int p_idx, RID *p_instances) {
-	_update_dirty_instance(p_idx, p_instances);
+	_update_dirty_instance(p_idx, p_instances, true);
 }
 
-void RaycastOcclusionCull::Scenario::_update_dirty_instance(int p_idx, RID *p_instances) {
+void RaycastOcclusionCull::Scenario::_update_dirty_instance(int p_idx, RID *p_instances, bool is_threaded) {
```

Comment:
How does this fix anything? It isn't used anywhere , this can't change any behavior